lilypond/songs/silent-night-holy-night/silent-night-holy-night.ly

```diff
@@ -0,0 +1,94 @@
+\version "2.22.1"
+#(ly:set-option 'relative-includes #t)
+\include "../../lib/hymn-common.ily"
+
+%% TUNE INFO
+composer = \smallText "Music: Franz Gruber, 1818"
+meter = \smallText "Irregular"
+hymnKey = \key bf \major
+hymnTime = \time 6/8
+hymnBaseMoment = \set Timing.baseMoment = #(ly:make-moment 1/8)
+hymnBeatStructure = \set Timing.beatStructure = 3,3
+quarternoteTempo = 50
+\include "../../lib/global-parts.ily"
+
+%% SONG INFO
+title = \titleText "Silent Night, Holy Night"
+poet = \twoLineSmallText "Text: Joseph Mohr, 1818;" "tr. John Freeman Young, 1863"
+typesetter = "Zachary Cline"
+verseCount = 1
+tags = "english christian christmas 4part"
+dateAdded = "2022-05-13"
+\include "../../lib/header.ily"
+
+%% NOTES
+soprano = {
+  \globalParts
+  \relative g' {
+    f8.( g16) f8 d4. | f8.( g16) f8 d4. | c'4 c8 a4. | bf4 bf8 f4. |
+    g4 g8 bf8.( a16) g8 | f8.\( g16\) f8 d4. | g4 g8 bf8.\( a16\) g8 | f8.\( g16\) f8 d4. |
```

Comment:
Well done with the \( 👏🏼

Reply:
Thanks! How to do this was well documented in the project.